frontend/src/ga-screen/ga-screen.vue

```diff
@@ -139,30 +152,39 @@ export default {
         let activeCitizen = this.get_citizen_for_csr(csr)
 
         if (activeCitizen === null) {
-          csr['wait_time'] = null
+          csr.csr_state.csr_state_name == 'Break' ? csr['wait_time'] = 'ON BREAK' : csr['wait_time'] = null;
```

Comment:
Nice ternary operator. This form is preferable though:
csr.wait_time = (csr.csr_state.csr_state_name === 'Break') ? 'ON BREAK' : null;